### DISCUSSION TITLE
Does this package REPLACE template instead of BUILD a docx file?

### DISCUSSION THREAD

**Discussion by weilies** [dolanmiu/docx#927] [Ideas / Requests]
Greeting gurus,

I am new to this package. I have glance through the examples but still doesn't get my answer.
https://github.com/dolanmiu/docx/blob/master/demo/30-template-document.ts

Does this package similar to following two? REPLACE the template variable and generate a complete .docx word doc?

- ttps://github.com/guigrpa/docx-templates
- https://github.com/open-xml-templating/docxtemplater

Instead of generate the whole doc via code?

My use case is simple, user simply upload a word doc (template with few template variables) 
![image](https://user-images.githubusercontent.com/484828/116856509-d612e380-ac2d-11eb-96a8-7dcd9fbcb02f.png)

Thanks for the input

**Comment by dolanmiu** [comment]
Nope

this package does not use any templates

This package builds a Word document from scratch using TypeScript.

**Reply by weilies** [reply]
Thanks for the confirmation!
